helm/defectdojo/templates/django-service.yaml

```
@@ -35,8 +35,8 @@ spec:
     port: 9113
     targetPort: http-metrics
     {{- end }}
-{{- if .Values.django.serviceType }}
-  type: {{ .Values.django.serviceType }}
+{{- if .Values.django.service.type }}
+  type: {{ .Values.django.service.type }}
 {{- end }}
 {{- if .Values.gke.useGKEIngress }}
```

Thanks for fixing this. Since these `type`s will conflict if `django.service.type` is set and `gke.useGKEIngress` is `true`, the GKE ingress check should be an `else if` instead of a separate `if`.

```suggestion
{{- else if .Values.gke.useGKEIngress }}
```